artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/FileLockNodeManager.java

```diff
@@ -377,7 +415,7 @@ protected FileLock lock(final int lockPosition) throws Exception {
                   return null;
```

Comment:
why are we catching the InterruptedException here and throwing it at the end ?

Reply:
Good point, let me check why I've made this change so I can comment :+1:

Reply:
I see why now, the original loop looked not correct...
the while (!interrupted) loop can end only if `interrupted == true`  and by contract we should throw `InterruptedException` for any caller there...but maybe the caller already take care of this, so better that I double check if I've changed semantic here, thanks!

Reply:
I've decided to not change the original loop here :+1: